src/redux/admin/admin.types.js

```diff
@@ -0,0 +1,6 @@
+export const SET_AUTH = 'SET_AUTH';
+export const SET_ADMIN_LOADING = 'SET_ADMIN_LOADING';
+export const SET_ADMIN_ERROR = 'SET_ADMIN_ERROR';
+export const LOGIN_ADMIN = 'LOGIN_ADMIN';
```

Comment:
There is no need to have `ADMIN` in the end. In admin application, only admins will be able to log in